src/data.js

```diff
@@ -19,5 +19,6 @@ module.exports.Build = Entity.configure({
     state: Entity.types.String,
     created: Entity.types.Date,
     updated: Entity.types.Date,
+    installationID: Entity.types.Number,
```

Comment:
If adding a new column to an Azure table, you'll need to [make a new "version" of the table](https://github.com/taskcluster/azure-entities#migrations), rather than just changing it in place.